### DISCUSSION TITLE
Accelerate model building in Yalmip

### DISCUSSION THREAD

**Discussion by BYS543** [yalmip/YALMIP#998] [General discussion]
Hi Jodan,
sometimes,i find it is slow to construct a model in yalmip when using for loop. I find matlab can use `parfor` to parallel. Can I use it to build models in parallel？

**Comment by johanlofberg** [comment]
In 99% of the cases you should vectorize. parfor will not help

**Reply by BYS543** [reply]
Ok,thank you,Johan.
